src/Query/ResultPrinters/TableResultPrinter.php

```diff
@@ -347,6 +350,34 @@ protected function getCellContent( array $dataValues, $outputMode, $isSubject )
 		return $html;
 	}
 
+	/**
+	 * @param @param SMWDataValue $value
+	 * @param int $outputMode
+	 * @param Linker|null|bool $linker
+	 * @return string
+	 */
+	protected function getDataValueText( $value, $outputMode, $isSubject ) {	
```

Comment:
Please us PHP types, not doc types. Yes, SMW mostly has types in docs, but that is because that code was written in the PHP 5.x days.

Reply:
Btw, any reason for this to be protected? Typically something should be private unless a collaborator needs it to be public. Protected is rarely the right choice.

Reply:
this method is now removed (I think I adopted the same access declaration from the caller method)